integration_tests/src/test/scala/com/nvidia/spark/rapids/tests/CacheTestSuite.scala

```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright (c) 2019-2020, NVIDIA CORPORATION.
```

Comment:
Was this written partly last year?

Reply:
I copied the license from a file that was. 

I have fixed the date. Please let me know if you have more comments